enclave/core/sgx/globals.c

```diff
@@ -145,24 +145,27 @@ const void* __oe_get_enclave_base()
 {
     return (uint8_t*)&_enclave_rva - _enclave_rva;
 }
-const void* __oe_get_enclave_Zero_base_start_address()
+
+size_t __oe_get_enclave_size()
 {
-    return oe_enclave_properties_sgx.start_addr;
+    return oe_enclave_properties_sgx.image_info.enclave_size;
 }
 
-unit8_t oe_get_enclave_create_zero_base_flag()
+const void* __oe_get_enclave_elf_header(void)
 {
-    return oe_enclave_properties_sgx.flags.create_zero_base_enclave;
+    return __oe_get_enclave_base();
 }
 
-size_t __oe_get_enclave_size()
+const void* __oe_get_enclave_zero_base_start_address()
 {
-    return oe_enclave_properties_sgx.image_info.enclave_size;
+    return oe_enclave_properties_sgx.start_addr;
 }
 
-const void* __oe_get_enclave_elf_header(void)
+unit8_t oe_get_enclave_create_zero_base_flag()
```

Comment:
```suggestion
unit8_t __oe_get_enclave_create_zero_base_flag()
```
Follow local consistency